src/reference/sectionC_interpretation.inc

```diff
@@ -281,9 +286,9 @@ Interpretation of ``initial_value`` attributes
 
 #. The :code:`initial_value` attribute of a :code:`variable` element MUST either be a :ref:`real number string<specA_real_number>`, or a :ref:`variable reference<specC_variable_reference>`.
 
-#. The conditions when initial values hold are (by design) not defined in a :ref:`CellML model<specA_cellml_model>` document.
+#. The conditions when initial values hold are (by design) not defined in a :ref:`CellML model<specA_cellml_model>`.
```

Comment:
Suggest: The conditions under which initial values hold ...
Dunno why, something about this bit sounds odd.

Reply:
Eh, but then we use "when" below too ... maybe just ignore me!

Reply:
I suppose "when x = 4" is still a "when"?

Reply:
Yes, when x = 4 is fine, but "conditions when" sounds odd.  Like I said, I don't know why!

Reply:
I prefer @kerimoyle's suggestion, whatever the implications on the rest of the text.

Reply:
Have committed a change!